contracts/test/TokenStakingTestSet.sol

```diff
@@ -182,6 +182,14 @@ contract NuCypherTokenStakingMock is INuCypherStakingEscrow {
     {
         return stakers[staker].value;
     }
+
+    function stakerInfo(address staker)
+        public
+        view
+        returns (StakerStruct memory)
+    {
+        return stakers[staker];
+    }
```

Comment:
Actually, one q: given that `stakers` is public, can't we query the contract with `contract.stakers(address)`?